src/renderer/components/FtIconButton/FtIconButton.scss

```
@@ -150,7 +150,10 @@
   position: absolute;
   text-align: center;
   user-select: none;
-  z-index: 3;
+  // Above route headers and the tab bar, while the top navigation stays above
```

Stylelint (17.14.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)